mh/database.py

```diff
@@ -345,6 +345,14 @@ def __init__(self):
             "C9I7D4": {"name": "Cid", "session": None},
             "D9R7K4": {"name": "Drakea", "session": None},
         }
+        self.friends_lists = {
```

Comment:
Since when the client request his friends, it require a index for each of them. You must use the `Players` collection, so friend's index stay stable over removal or addition.